src/en/darthsdroids/src/eu/kanade/tachiyomi/extension/en/darthsdroids/DarthsDroids.kt

```diff
@@ -0,0 +1,222 @@
+                        name = a.text()
+                        chapter_number = i.toFloat()
+                        date_upload = DATE_FMT.parse(pageData[0].text().trim())?.time ?: 0L
+                        url = absolutePageUrl(a.attr("href"), archivePath)
+                    },
+                )
+                i += 1
+            } else if (!pageData.hasAttr("colspan")) {
+                // Are we in a dateless archive?
+                pageAnchor = pageData.getOrNull(0)?.selectFirst("a")
+                if (pageAnchor != null) {
+                    // For now we assume all dateless archives were published the same day
+                    // the Solo book was published, because currently Solo’s the only one such book.
+                    val pageDate = 1660435200000L + (i * 1000 * 60)
+                    val a = pageAnchor
+
+                    chapters.add(
+                        SChapter.create().apply {
+                            name = a.text()
+                            chapter_number = i.toFloat()
+                            date_upload = pageDate
+                            url = absolutePageUrl(a.attr("href"), archivePath)
+                        },
+                    )
+                    i += 1
+                }
+            }
+        }
+        chapters.reverse()
+        return chapters
+    }
+
+    private fun absolutePageUrl(pageUrl: String, maybeRelativeTo: String): String =
+        if (pageUrl.startsWith('/')) {
+            pageUrl
+        } else {
+            // This happens because for whatever reason, the `solo` index isn’t
+            // a `.html` page, but a directory path with an `index.html`. So
+            // we’re returning `solo/$pageUrl`.
+            maybeRelativeTo + pageUrl
+        }
+
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        var archiveResponse = client.newCall(GET(baseUrl + manga.url, headers)).execute()
+        if (!archiveResponse.isSuccessful) {
+            archiveResponse = client.newCall(GET(mainArchiveUrl, headers)).execute()
+        }
+
+        // Fingers crossed they don’t come up with yet another different archive structure.
+        // Technically `manga.url` is »incorrect« for the main archive, but that parameter
+        // is ignored for almost all archives anyway. Only those not pointing at a `.html`
+        // page matter.
+        return Observable.just(fetchChaptersForDatedAndDatelessArchives(archiveResponse.asJsoup(), manga.url))
+    }
+
+    override fun fetchPageList(chapter: SChapter): Observable<List<Page>> =
+        Observable.just(listOf(Page(0, chapter.url)))
+
+    override fun fetchImageUrl(page: Page): Observable<String> {
+        val comicPage = client.newCall(GET(baseUrl + page.url, headers)).execute().asJsoup()
+        // Careful. For almost all images it’s `div.center>p>img`, except for pages released on
+        // April’s Fools day, when it’s `div.center>p>a>img`.
```

Comment:
Since you're documenting this behavior anyways, please also include a link to where the default assumption breaks.